meetup/models.py

```diff
@@ -0,0 +1,40 @@
+from django.db import models
+from pizzaplace.models import PizzaPlace
+from django.core.validators import RegexValidator
+from meetup.services.meetup_api_lookup_agent import MeetupApiLookupAgent
+from django.core.exceptions import ValidationError
+
+
+def validate_urlname(link):
+  validator = RegexValidator(
+    regex='meetup\.com\/\w+(-\w+)*\/$',
+    message="Does not conform to Meetup Url",
+    code='invalid_url')
+  return validator(link)
+
+def validate_meetup_exists(link):
+  looker = MeetupApiLookupAgent(link)
+  is_real = looker.is_real_meetup()
+
+  if not is_real:
+    raise ValidationError("That's not a meetup")
```

Comment:
Similar to the regex message above, is this a useful error? If this throws unexpectedly and you discover this exception in some log file two weeks from now, will it give you enough information and context to figure out what went wrong?